governance/polls/Vault Compensation Denomination - April 13, 2020.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Vault Compensation Denomination Poll - April 13, 2020
+summary: Signal whether to provide compensation to vault holders in the vault's token (ETH/BAT) or in DAI
+discussion_link: https://forum.makerdao.com/t/1822
+poll_rules: The voter may select to vote for one of the poll options or they may elect to abstain from the poll entirely
+options:
+   0: Abstain
+   1: the asset that was liquidated (ETH or BAT)
```

Comment:
small suggestion, but could we change to the following? I reads a bit simpler I think.
```suggestion
   1: ETH or BAT (the asset that was liquidated) 
```